doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -217,23 +222,28 @@ To create an nRF Cloud account, complete the following steps:
    If you closed the registration dialog box, you can repeat Step 1 and then click :guilabel:`Already have a code?`.
    Then enter your email and the verification code.
 
-You can now log in with your email and the password you chose.
+You can now log in to `nRF Cloud`_ with your email and password.
 After logging in, you are taken to the dashboard view that displays your device count and service usage.
+Next, you need to activate the SIM card for the |DK|.
```

Comment:
```suggestion
Next, you need to activate the SIM card you will use in the |DK|.
```
Current version can be interpreted like the SIM card needs to be activated specifically for the DK, which is not the case.